src/ShallowTraversal.js

```diff
@@ -151,3 +154,21 @@ export function getTextFromNode(node) {
     .join('')
     .replace(/\s+/, ' ');
 }
+
+export const findRefInOwnedChildren = (node, ref, isRootNode) => {
```

Comment:
Yeah, good idea. I'm on it. Thanks for the feedback btw. I haven't had the chance to have my code reviewed much.